.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,52 +1,7 @@
-<!--
+## Description
+<!-- Describe the changes. Link to any issues closed by this pull request. -->
```

Comment:
```suggestion
<!-- Standalone description of changes (will be included in CHANGELOG). Reference any issues closed by this PR with "closes #1234" or "fixes #5678" -->
```

Reply:
Let's be consistently imperative here:

```
<!-- Provide a standalone description of changes in this PR (will be included in CHANGELOG). -->
<!-- Reference any issues closed by this PR with "closes #1234". -->
```

Is the description for each PR actually included in CHANGELOG? I don't see them: https://github.com/rapidsai/cudf/blob/branch-22.06/CHANGELOG.md

Reply:
You're correct. The _title_ is included in the changelog. I do think that's helpful to note.

Reply:
Updated in 2cf27046840523ed61471bb981fe23c8d96551e5. Feel free to open a new thread for further suggestions. This one is outdated so I'll resolve it.